menpo/io/input/base.py

```diff
@@ -596,6 +596,9 @@ def importer_for_filepath(filepath, extensions_map, importer_kwargs=None):
     """
     suffix = ''.join(filepath.suffixes)
     importer_type = extensions_map.get(suffix)
+    if (importer_type is None) and len(filepath.suffixes) > 1:
+        suffix = ''.join(filepath.suffix)
```

Comment:
This seems like a good place to leave a little comment to make it clear what we are doing:
```python
# we couldn't find an importer for all the suffixes (e.g .foo.bar)
# maybe the file stem has '.' in it? -> try again but this time just use the
# final suffix (.bar). (Note we first try '.foo.bar' as we want to catch
# cases like 'pkl.gz')
```